docs/02.API-REFERENCE.md

```diff
@@ -5523,6 +5531,63 @@ jerry_get_backtrace (uint32_t max_depth);
 
 - [jerry_create_external_function](#jerry_create_external_function)
 
+## jerry_get_api_major_version
+
+**Summary**
+
+Get the major API version.
+
+**Prototype**
+
+```c
+uint32_t
+jerry_get_api_major_version(void);
+```
+
+- return value
+  - API major version
+
+## jerry_get_api_minor_version
+
+**Summary**
+
+Get the minor API version.
+
+**Prototype**
+
+```c
+uint32_t
+jerry_get_api_minor_version(void);
+```
+
+- return value
+  - API minor version
+
+## jerry_get_component_version
```

Comment:
If we have this method, why do we need the previous two? (get_api_minor & get_api_major)?